src/libsystemd/libsystemd.sym

```diff
@@ -824,4 +824,5 @@ global:
         sd_event_source_set_memory_pressure_period;
         sd_event_trim_memory;
         sd_pid_notify_barrier;
+        sd_event_source_leave_ratelimit;
```

Comment:
Uhhh this needs to be moved to v254

Reply:
it *is* in the 254 section. (this is confusing, indeed, but open the context of this patch and you'll see we define this in the 254 section and then chain it up with the 253 section)